dGame/dComponents/RacingControlComponent.cpp

```diff
@@ -99,7 +98,9 @@ void RacingControlComponent::LoadPlayerVehicle(Entity* player,
 
 	if (item == nullptr) {
 		Game::logger->Log("RacingControlComponent", "Failed to find item");
-
+		auto* playerInstance = dynamic_cast<Player*>(player);
```

Comment:
I know its not done above, but when doing a `dynamic_cast` from one pointer to another, you generally want to check that the resulting pointer is not null.  Should the dynamic_cast fail, this would be a nullptr and crash.  It wouldn't happen in practice, but always good to check your bounds.
c.f. https://en.cppreference.com/w/cpp/language/dynamic_cast section 5.c